packages/plugin-corsi-blocks/examples/example.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <script src="https://unpkg.com/jspsych@7.3.2"></script>
+    <script src="https://unpkg.com/jspsych@8.0.2"></script>
```

Comment:
Is this the desired version? 8.0.2?

Reply:
it was the most recent version at the time of writing; though we could get around this by simply doing ``<script src="https://unpkg.com/jspsych@8"></script>`` to always query the latest version?